### DISCUSSION TITLE
Additional commands?

### DISCUSSION THREAD

**Discussion by aneisch** [wimaha/TeslaBleHttpProxy#64] [Q&A] [answered]
I've just discovered this project and am still working to understand things. The readme suggests that only a limited subset of commands are supported versus what [tesla-control](https://github.com/teslamotors/vehicle-command/tree/main/cmd/tesla-control) offers. 

Is this still true, or is there parity between the available commands (eg: seat heat, climate, or media controls)?

My goal is to have an http API that can send BLE commands to my vehicle, essentially wrapping `tesla-control`. I'm trying to determine if this is it or if I need to build something of my own (or contribute here!)

**Comment by wimaha** [comment, marked as answer]
I think you can use this project for you needs. Currently not all commands are available. But it is very easy to add missing commands. So either make a PR or create a new issue describing which commands are needed.

I have recently added the possibility to fetch vehicle information (ex. SOC) via the the proxy via BLE.

**Reply by aneisch** [reply]
Thank you! After some more review it does look like it's pretty trivial to add new commands. Once my raspberry pi arrives I suspect I'll have some PRs to submit. Thanks!
